### PR TITLE
Initial structure

### PR DESCRIPTION
**Pull request by adrianovi**

### CODE REVIEW

**Review by eliosfg — APPROVED** [review]
Let's go ahead

**Review by jimytz — APPROVED** [review]
👍

File: src/main/java/framework/CredentialsManager.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    public String getUserName(String userRole) {
+        return getEnvironmentSetting("userRole.username".replace("userRole", userRole));
+    }
+
+    public String getPassword(String userRole) {
+        return getEnvironmentSetting("userRole.password".replace("userRole", userRole));
+    }
+
+    public int getAPIServicePort() {
```
Comment (Agustin-Mediotti) [inline src/main/java/framework/CredentialsManager.java:93]: This is not needed

File: src/main/java/api/APIManager.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    private void initialize() {
+        log.info("Initializing API Manager");
+        RestAssured.baseURI = CredentialsManager.getInstance().getBaseURL();
+        RestAssured.basePath = CredentialsManager.getInstance().getBasePath();
+        RestAssured.port = CredentialsManager.getInstance().getAPIServicePort();
+    }
+
+    public void setCredentials(String userName, String password) {
```
Comment (Agustin-Mediotti) [inline src/main/java/api/APIManager.java:35]: This is not needed

File: src/main/java/api/APIManager.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
+package api;
+
+import framework.CredentialsManager;
+import io.restassured.RestAssured;
+import io.restassured.authentication.PreemptiveBasicAuthScheme;
```
Comment (Agustin-Mediotti) [inline src/main/java/api/APIManager.java:5]: Also remember to delete the unused import

**Review by Agustin-Mediotti — DISMISSED** [review]
Delete the unused methods

File: src/main/java/api/APIManager.java (hunk 3 above)
Comment (adrianovi) [reply src/main/java/api/APIManager.java]: Good observation, I fixed it. The port I think we accorded to leave it just in case

**Review by SerJimmy — APPROVED** [review]

**Review by Agustin-Mediotti — APPROVED** [review]

**Review by jimytz — APPROVED** [review]
👍

**Review by eliosfg — APPROVED** [review]
LGTM!

**Pull request merged**